beater/beater.go

```diff
@@ -3,37 +3,37 @@ package beater
 import (
 	"fmt"
 
-	"github.com/elastic/apm-server/config"
 	"github.com/elastic/apm-server/server"
 	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 )
 
-type ApmServer struct {
+type ApmBeater struct {
```

Comment:
We could also call this just `Apm` or only `Beater`. I'm slightly in favor of `Apm`.